eclass/tree-sitter-grammar.eclass

```diff
@@ -42,13 +42,7 @@ EXPORT_FUNCTIONS src_compile src_install
 # This internal function determines the ABI version of a grammar library based
 # on the package version.
 _get_tsg_abi_ver() {
-	if ver_test -gt 0.21; then
-		die "Grammar too new; unknown ABI version"
-	elif ver_test -ge 0.19.0; then
-		echo 13
-	else
-		die "Grammar too old; unknown ABI version"
-	fi
+	sed -r -n '/#define LANGUAGE_VERSION ([0-9]*)$/{s@@\1@;p}' "${S}"/parser.c
```

Comment:
Can we at least add a comment explaining the regex? It's obviously much less readable than ver_test

Reply:
@00-matt said that regex failed for him and suggested another way (which is also much more readable), I'll test and apply it in meantime.